pkg/dependencymagnet/dependencymagnet.go

```diff
@@ -0,0 +1,11 @@
+// +build tools
```

Comment:
what is this? some sort of dependency injection?

Reply:
not exactly - we use it so that some useful shell scripts show up in `vendor` and `go mod` won't prune them.